internal/forest/forest.go

```diff
@@ -95,7 +99,7 @@ func (f *Forest) Get(nm string) *Namespace {
 		name:          nm,
 		children:      namedNamespaces{},
 		sourceObjects: objects{},
-		quotas:        quotas{limits: limits{}, used: usage{}},
+		quotas:        make(map[RQName]*quotas),
```

Comment:
Hint for reviewers:
This is the fairly large change we are making in the first commit. 089f143b534b5c7d7a86bac2619c57836b84ad8d

Until now, we only had HRQ without Scope, so we could manage everything with a single quota, aka SingleResourceQuota. However, with supporting Scoped HRQ, that is no longer possible, so we have changed the data structure.

The RQName will contain the name of the RQ generated from the Singleton and each HRQ that has a Scoped, as before.

For example, in the case of HRQ-not-having-scope-A / HRQ-not-having-scope-A / Scoped-HRQ-A / Scoped-HRQ-B, the quotas will have three keys.